app/boards/arm/nice_nano/nice_nano.dtsi

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/dts-v1/;
```

Comment:
As I look at this, we might be doing this elsewhere already for splits, but I wonder if this early directive should be kept in the toplevel `dts` file, not in any sub-includes. Thoughts?

Reply:
Looking at Zephyr, they seem to have this in both `dtsi`s and `dts` files for some boards and just in the `dts` for others. Most seem to use just once in the `dts`, so I'll go that route.